### ISSUE TITLE
Kibana 7.2.0

### ISSUE THREAD

**Issue by lucasapaixao** [datasweet/kibana-datasweet-formula#52]
To use the plugin in Kibana 7.2.0 i modify the package.json and it was possible to use the plugin.
But now i have a problem with Metric visualization, when i "Hide Aggregation" Kibana produces a error.
The Datasweet formula is simples: agg1/agg2
The Kibana error:
"Version: 7.2.0
Build: 24337
Error: Uncaught TypeError: Cannot read property 'name' of undefined (http://kibana.unimedva.com.br/built_assets/dlls/vendors.bundle.dll.js:98)
    at window.onerror (http://kibana.unimedva.com.br/bundles/commons.bundle.js:3:1643907)"

**Comment by karthichana** [comment]
I'm having the same issue when I tried to 'Hide Aggregation' under Kibana 7.1.1.

**Comment by lfroment-datasweet** [comment]
Hi all, 7.2 is not supported yet. under 7.1.1 OSS, bug cannot be reproduced. Everything is working fine. @karthichana , what version are you using, and what other plugins have you installed?
![711_oss_hide_no_bug](https://user-images.githubusercontent.com/20048888/61947783-68d91580-afa6-11e9-82a3-0371483e1458.png)

**Comment by karthichana** [comment]
@lfroment-datasweet it's 7.11 and you refer to the error message triggered when i tried to do the hiding.

![image](https://user-images.githubusercontent.com/8412632/62114956-37d04c00-b2ea-11e9-8254-ec0749a67343.png)

**Comment by tonykoval** [comment]
Here is list of API changes in 7.2, https://www.elastic.co/blog/kibana-plugin-api-changes-in-7-2

**Comment by tonykoval** [comment]
7.3, https://www.elastic.co/blog/kibana-plugin-api-changes-in-7.3

**Comment by joshkleven09** [comment]
@lfroment-datasweet I am seeing this same issue as well when using Kibana v7.1.1. Hiding the aggregation works fine in Data Table visualizations (like your example) but I get the same error posted above when trying to hide an aggregation in a Metric visualization.

**Comment by karthichana** [comment]
@joshkleven09 Yes you are right, it's ok with the  Data Table visualizations but it doesnt work in the Metric visualization.

**Comment by lfroment-datasweet** [comment]
Hi, 
got it. Indeed, we have the same issue with OSS version. We have to investigate. It may take some time beacause we are pretty busy + vacations.
Best regards to you all!

**Comment by guanghaofan** [comment]
@lfroment-datasweet 
I think it needs some time to fix the issues in v7.1.x and the other upstream versions due to the big changes in the aggconfig API. and we really hope that the new version can be released soon. You guys did grate job! thank you!
